java/src/test/java/com/saucelabs/simplesauce/SauceOptionsTest.java

```diff
@@ -1,31 +1,92 @@
 package com.saucelabs.simplesauce;
 
+import org.junit.Ignore;
 import org.junit.Test;
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.BrowserType;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+import static org.openqa.selenium.UnexpectedAlertBehaviour.DISMISS;
 
-public class SauceOptionsTest {
-    private SauceOptions options = new SauceOptions();
+public class SauceOptionsTest extends BaseConfigurationTest{
+    @Test
+    public void usesLatestChromeWindowsVersions() {
```

Comment:
@titusfortner there are 3 tests that already cover this.
https://github.com/saucelabs/simple_sauce/blob/46abed3f500d282bbe4d5796f4fe8ecc2dd85ae9/java/src/test/java/com/saucelabs/simplesauce/SauceOptionsTest.java#L11

Reply:
I'll look, I did a lot of rebasing and might have left in some duplicates.